gulpfile.js

```diff
@@ -0,0 +1,44 @@
+var gulp                = require('gulp'),
+    livereload          = require('gulp-livereload'),
+    nodemon             = require('gulp-nodemon'),
+    shell               = require('gulp-shell'),
+    chalk               = require('chalk'),
+    paths,
+    nodemonServerInit;
+
+paths = {
+    server: 'core/server/',
+    client: 'core/client/'
+};
+
+nodemonServerInit = function () {
+    livereload.listen();
+
+    return nodemon({
+        script: 'index.js',
+        ext: 'js,json',
+        watch: [
+            'core/index.js',
+            paths.server + '**/*.js',
+            'core/built/assets/*.js'
+        ],
+        ignore: [
+            'core/client/*',
+            'core/server/test/*'
+        ]
+    }).on('restart', function () {
+        console.info(chalk.cyan('Restarting Ghost due to changes...'));
+        gulp.src('index.js')
+            .pipe(livereload());
+    });
+};
+
+// Starting Ember for development
+gulp.task('admin', shell.task(['cd ' + paths.client + ' && ember build --watch']));
+
+gulp.task('serve', function () {
```

Comment:
Silly little thing, but if we made this "server" instead of "serve" then we would be able to run either:

`gulp dev` (to do all)
`gulp admin` (just the admin panel)
`gulp server` (just the server)

I think?